rocket/todo-postgres/README.md

```diff
@@ -0,0 +1,7 @@
+# Hello World Application using Post-Gres
+The following example is to present a way to use [rocket](https://rocket.rs/v0.4/guide/introduction/) framework and devlop a relational database for a TODO app using [Postgres](https://www.postgresql.org/docs/).
```

Comment:
See the review on the poem postgres readme, the same applies here. We should also link to the rocket home page.